### ISSUE TITLE
Ignore non-recognized markdown contents

### ISSUE THREAD

**Issue by soubinan** [stateful/vscode-runme#1455]
Hi,

Firstly, thank you for this great tool!

I am using it for weeks now, and I noticed an annoying issue.

When you have a markdown file with some exotic content (like presentations in Markdown format in instance)
It will be great if RUNME could just ignore them (or raise a notification we can close) and let us read the markdown file anyway (without the RUNME parsing)
Currently, RUNME shows an error instead of showing the file content.
To have a kind of fallback behaviour will be nice, so the non-recognized Markdown format would still be usable in VS Code.

Below, a screenshot to illustrate my experience:

![image](https://github.com/stateful/vscode-runme/assets/20760371/6fe58850-f203-4ded-ab61-dad2d756d72f)

In this example, the Markdown is in a special format for Hugo:

```sh
$ cat archetypes/default.md 

+++
title = '{{ replace .File.ContentBaseName "-" " " | title }}'
date = {{ .Date }}
draft = true
+++
```

**Comment by sourishkrout** [comment]
Thanks for reporting this issue, @soubinan. Runme's deserializer should indeed be able to ignore+retain the content. We'll look into it. Will keep you posted.

**Comment by sourishkrout** [comment]
@soubinan this is fixed as of [v3.6.2](https://github.com/stateful/vscode-runme/releases/tag/3.6.2) (just updated to latest in the vscode). Lemme me know how it goes please.

**Comment by soubinan** [comment]
Cool! It works as expected!
Thank you
